less/mixins.less

```diff
@@ -1 +1,24 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+
+.custom-button(@width: 50px, @height: 50px, @bcolor: white, @fcolor: black) {
+  width: @width;
+  height: @height;
+  background-color: @bcolor;
+  color: @fcolor;
+  font-size: 1.8rem;
+  border-radius: 15px;
+  cursor: pointer;
+
+  &:hover {
+    background-color: @fcolor;
+    color: @bcolor;
+  }
+}
```

Comment:
looking good so far, just remember the `height` shouldn't need to be hard-coded in at all since you want to control that with vertical padding instead. 4 params into one parametric mixin is also a lot (it still works), only include that height/vertical padding control if you think you'll need it. nice use of the default values though